query/test/unit/api_server/test_endpoints.py

```diff
@@ -0,0 +1,31 @@
+import json
+import os
+import sys
+import unittest
+
+from unittest.mock import patch
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '../../../..'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from query.test.unit.api_server import client_for_test_api_server
+
+
+class TestEndpoints(unittest.TestCase):
+
+    def test_heathcheck_endpoint(self):
+        self.client = client_for_test_api_server()
+
+        response = self.client.get(f"/v1/health")
+
+        self.assertEqual(response.status_code, 200)
```

Comment:
format: spacing is a little different here relative to the other test.

Reply:
👍